.github/workflows/e2e-cleanup.yml

```
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: blackbaud/skyux-pr-preview
-          ref: 'main'
+          ref: '12.x.x'
```

This is the GH pages repo. It only uses `main` and should not be updated.
```suggestion
          ref: 'main'
```